src/server/api/routes/reader/index.js

```diff
@@ -0,0 +1,60 @@
+const { Router } = require('express');
+const {
+  apiFetchLikedUser,
+  apiFetchUserArticles,
+  apiFetchSuggestedArticles,
+} = require('../../util/api');
+const { userCollection } = require('../../util/firebase');
+const follow = require('./follow');
+
+const router = Router();
+
+router.use(follow);
+
+router.get('/reader/index', async (req, res, next) => {
+  try {
+    if (!req.session.user) {
+      res.sendStatus(403);
+      return;
+    }
+    const [{ data }, userDoc] = await Promise.all([
+      apiFetchLikedUser(req),
+      userCollection.doc(req.session.user).get(),
+    ]);
+    const userSet = new Set(data.list);
+    const { followedUsers = [], unfollowedUsers = [] } = userDoc.data();
+    followedUsers.forEach(u => userSet.add(u));
+    unfollowedUsers.forEach(u => userSet.delete(u));
+    res.json({ list: Array.from(userSet), unfollowedUsers });
```

Comment:
Discussion: Is it necessary to return unfollowedUsers?

Reply:
yes, if you want to un-unfollow